React/CoreModules/RCTDevLoadingView.mm

```diff
@@ -107,17 +107,20 @@ - (void)showMessage:(NSString *)message color:(RCTUIColor *)color backgroundColo
       self->_label.textAlignment = NSTextAlignmentCenter;
 #elif TARGET_OS_OSX // [TODO(macOS ISS#2323203)
       NSRect screenFrame = [NSScreen mainScreen].visibleFrame;
-      self->_window = [[NSPanel alloc] initWithContentRect:NSMakeRect(screenFrame.origin.x + round((screenFrame.size.width - 375) / 2), screenFrame.size.height - 22, 375, 22)
+      self->_window = [[NSPanel alloc] initWithContentRect:NSMakeRect(screenFrame.origin.x + round((screenFrame.size.width - 375) / 2), screenFrame.size.height - 19 - 1, 375, 19)
```

Comment:
Just make the -19 -1 into -20?

Reply:
Sure, I just wanted to avoid 20 being a magic number, hence the extra -1. Shall I add a comment that the n-1 offset is intentional or should I assume it's clear enough from the commit/PR message?